src/pages/[product]/tutorials/[collectionSlug].tsx

```diff
@@ -0,0 +1,7 @@
+import CollectionView from 'views/collection-view'
+import { generateStaticFunctions } from 'views/collection-view/server'
+
+const { getStaticProps, getStaticPaths } = generateStaticFunctions()
+
+export { getStaticPaths, getStaticProps }
```

Comment:
I think defining `getStaticProps` and paths would be useful here now. Otherwise this file doesn't really have meaningful data.